paasta_tools/check_marathon_services_replication.py

```diff
@@ -158,6 +158,33 @@ def check_smartstack_replication_for_instance(
 
         if any(under_replication_per_location):
             status = pysensu_yelp.Status.CRITICAL
+            output += (
+                "\n\n"
+                "What this alert means:\n"
+                "\n"
+                "  This replication alert means that a SmartStack powered loadbalancer (haproxy)\n"
+                "  doesn't have enough healthy backends. Not having enough healthy backends\n"
+                "  means that clients of that service will get 503s (http) or connection refused\n"
+                "  (tcp) when trying to connect to it.\n"
+                "\n"
+                "Reasons this might be happening:\n"
+                "\n"
+                "  The service may simply not have enough copies or it could simply be\n"
+                "  unhealthy in that location. There also may not be enough resources\n"
+                "  in the cluster to support the requested instance count.\n"
+                "\n"
+                "Things you can do:\n"
+                "\n"
+                "  * Widen SmartStack discovery settings\n"
```

Comment:
I'd move this and 'increase the instance count' to a 'if your service isn't broken' section - this isn't the first thing to do, I wouldn't want people reading this output as an order of operation.